cmake/sip_helper.cmake

```diff
@@ -75,7 +75,14 @@ function(build_sip_binding PROJECT_NAME SIP_FILE)
 
     set(SIP_BUILD_DIR ${sip_BINARY_DIR}/sip/${PROJECT_NAME})
 
-    set(INCLUDE_DIRS ${${PROJECT_NAME}_INCLUDE_DIRS} ${PYTHON_INCLUDE_DIRS})
+    set(INCLUDE_DIRS)
+    foreach(dir ${${PROJECT_NAME}_INCLUDE_DIRS} ${PYTHON_INCLUDE_DIRS})
+      get_filename_component(dir_real "${dir}" REALPATH BASE_DIR ${sip_SOURCE_DIR})
+      # filter out /usr/include, which must not be included via -isystem
+      if (NOT "${dir_real}" STREQUAL "/usr/include")
```

Comment:
I don't understand why filtering `/usr/include` is necessary, but I'm willing to try it in Noetic and see if it causes issues.